src/WebJobs.Script/Description/DotNet/Compilation/Raw/RawAssemblyCompilation.cs

```diff
@@ -52,7 +52,7 @@ public FunctionSignature GetEntryPointSignature(IFunctionEntryPointResolver entr
                 throw new InvalidOperationException($"The function type name '{typeName}' is invalid.");
             }
 
-            MethodInfo method = functionType.GetMethod(methodName, BindingFlags.Static | BindingFlags.Public);
+            MethodInfo method = functionType.GetMethod(methodName, BindingFlags.Static | BindingFlags.Instance | BindingFlags.Public);
```

Comment:
Did you validate the behavior manually? This will likely not behave as expected as we don't manage or pass an instance when invoking the target.